settings/provisioner/virsh/virsh.yml

```diff
@@ -23,6 +23,7 @@ provisioner:
         - xorg-x11-apps
         - xauth
         - virt-viewer
+        - wget
```

Comment:
I don't think this should be part of this PR as this is "Beaker fix" not virsh, but it is definitely needed, though ;)